ner.py

```diff
@@ -75,13 +80,30 @@ def main():
     entity_types = set(entity_types)
     print("Extracting entities of types:", entity_types, "from '*.txt' documents in", args.data)
 
-    for path in glob.iglob(os.path.join(args.data, '*.txt')):
+    file_paths = glob.glob(os.path.join(args.data, '*.txt'))
+    num_files = len(file_paths)
+    print('Num files at path {}: {}'.format(args.data, num_files))
+    entities_for_file = {}
+    for idx, path in enumerate(file_paths):
         with open(path, 'r') as fhandle:
-            print("Processing:", fhandle.name)
-            doc = fhandle.read()
-            ents = extract_entities(doc, entity_types)
-            print("{}: {}".format(fhandle.name, ents))
+            try:
+                print("Processing:", fhandle.name)
+                doc = fhandle.read()
+                ents = extract_entities(doc, entity_types)
+                print("{}: {}".format(fhandle.name, ents))
+                entities_for_file[fhandle.name] = ents
+            except Exception as e:
+                print('Error reading file: \n {}'.format(e))
 
+        if idx % 100 == 0:
+            print('{}/{}')
```

Comment:
guess the values for the format string are missing

Reply:
Fixed, noticed it yesterday hahaha.